build.sbt

```diff
@@ -1,25 +1,30 @@
 import NativePackagerHelper._
 
-ThisBuild / scalaVersion     := "2.13.1"
-ThisBuild / version          := "0.1.0-SNAPSHOT"
-ThisBuild / organization     := "com.lunatech"
+ThisBuild / scalaVersion := "2.13.1"
```

Comment:
Nit, I'd just bump this to 2.13.3 right away.

Reply:
Same for the dependencies/plugins; might as well update all of them to their latest versions.